src/Sarif.FunctionalTests/ConverterTestData/StaticDriverVerifier/cancelspinlock_bug1.tt.sarif

```diff
@@ -7,7 +7,7 @@
         "name": "StaticDriverVerifier"
       },
       "files": {
-        "..\\..\\..\\rules\\WDM\\CancelSpinLock.slic": {
+        "rules/WDM/CancelSpinLock.slic": {
```

Comment:
I don't quite understand why we've lost the path traversal segments (the .. string). But if this is expected, we should probably add a path traversal string to the test cases.